### PR TITLE
Main site web app

### PR DESCRIPTION
**Pull request by karlTGA**
Add some buttons to the web app and a basic structur. 

image

### CODE REVIEW

File: vosekast_control/Vosekast.py [hunk 1]
```diff
@@ -317,12 +317,12 @@ def clean(self):
...
                         valve.close()
                         return
-                    if command['command'] == 'open':
+                    elif command['command'] == 'open':
                         valve.open()
                         return
                     else:
                         self.logger.warning(
-                            f'target_id found. command {command["command"]} did not execute.')
+                            f'did not found target_id {target_id}. command {command["command"]} did not execute.')
```
Comment (immunity-anomaly) [inline vosekast_control/Vosekast.py]: if it gets there, it has already passed the 
```
if valve.name == target_id:
```
check. therefore 
```
self.logger.warning(f'target_id found. command {command["command"]} did not execute.')
```
should be correct, but there was no matching command message.

File: vosekast_control/Vosekast.py [hunk 2]
```diff
@@ -342,7 +342,7 @@ def clean(self):
                         pump.toggle()
                     else:
                         self.logger.warning(
-                            f'target_id found. command {command["command"]} did not execute.')
+                            f'target_id not found. command {command["command"]} did not execute.')
```
Comment (immunity-anomaly) [inline vosekast_control/Vosekast.py]: see above

File: vosekast_control/Vosekast.py [hunk 3]
```diff
@@ -368,7 +368,7 @@ def clean(self):
                     #     return
                     else:
                         self.logger.warning(
-                            f'target_id found. command {command["command"]} did not execute.')
+                            f'target_id not found. command {command["command"]} did not execute.')
```
Comment (immunity-anomaly) [inline vosekast_control/Vosekast.py]: see above

File: vosekast_control/Vosekast.py [hunk 4]
```diff
@@ -391,7 +391,7 @@ def clean(self):
 
                 else:
                     self.logger.warning(
-                        f'target_id found. command {command["command"]} did not execute.')
+                        f'target_id not found. command {command["command"]} did not execute.')
```
Comment (immunity-anomaly) [inline vosekast_control/Vosekast.py]: see above

File: vosekast_control/Vosekast.py [hunk 5]
```diff
@@ -420,7 +420,7 @@ def clean(self):
...
```
Comment (immunity-anomaly) [inline vosekast_control/Vosekast.py]: see above

File: vosekast_control/Vosekast.py (hunk 1 above)
Comment (karlTGA) [reply vosekast_control/Vosekast.py]: correct i fixed that

**Pull request merged**